lib/jekyll/filters.rb

```diff
@@ -223,7 +224,9 @@ def group_by(input, property)
     def where(input, property, value)
       return input unless input.is_a?(Enumerable)
       input = input.values if input.is_a?(Hash)
-      input.select { |object| Array(item_property(object, property)).map(&:to_s).include?(value.to_s) }
+      input.select do |object|
+        Array(item_property(object, property)).map(&:to_s).include?(value.to_s)
```

Comment:
Sorry, didn't get it 😕 
You suggest to replace `Array(item_property(object, property)).map(&:to_s).include?(value.to_s)` line with `[item_property(object, property)].map(&:to_s).include?(value.to_s)` to make it shorter?